### ISSUE TITLE
No context for a certain string

### ISSUE THREAD

**Issue by yarons** [commons-app/apps-android-commons#2927]
https://translatewiki.net/wiki/Wikimedia:Commons-android-strings-review%20c%20violation%20report%20problem/he

I don't know what it means, how should I translate it? Is it OK to ask for a description?

**Comment by neslihanturan** [comment]
Hi, sorry I am not sure I understand properly. We have a string variable as: 
<string name="review_c_violation_report_problem">copyRightViolation</string>
Does your question is about this string?

**Comment by yarons** [comment]
Yes, should it be translated? Can I keep it like that? Should it even be here?

**Comment by neslihanturan** [comment]
No it shouldn't be there, it should be a constant in the code actually. Sorry for bothering, please leave it as is I will create an issue to remove it.

**Comment by yarons** [comment]
Sure, thank you :)

**Comment by amire80** [comment]
Not a string in the app.
